doc/en/components/charts/chart-types-bar.md

```diff
@@ -6,23 +6,118 @@ mentionedTypes: ["XamDataChart"]
+
+- Sales Management 
+- Inventory Management 
+- Stock Charts 
+- Any String Value Comparing a Numeric Value or Time-Series Value 
+
+## $PlatformShort$ Bar Chart Example
+
+In this example, the bar chart is comparing the total number of Olympic Medals (the Category) won between the USA and Russia from 1996 to 2016. The Y-Axis, or labels on the left of the chart, are displaying the time-series values of Year. The X-Axis, or labels on the bottom of the chart, are displaying the number of medals won.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Bar Chart Annotations
+
+In this example, the bar chart uses annotations to show a callout with the value for each bar.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
 
 ## $PlatformShort$ Stacked Bar Chart
 
-<!-- TODO copy and combine content (code snippets, description) from these topics:
-	data-chart-type-stacked-bar-series.md
-    data-chart-type-stacked-100-bar-series.md
--->
+A Stacked Bar Chart, or Stacked Bar Graph, is a type of category chart that is used to compare the composition of different categories of data by displaying different sized fragments in the horizontal bars of the chart. The length of each bar, or stack of fragments, is proportionate to its overall value. 
+
+The $PlatformShort$ Stacked Bar Chart differs from the $PlatformShort$ Bar Chart in that the data points representing your data are stacked next to each other horizontally to visually group data your data. Each stack can contain both positive and negative values. All positive values are grouped on the positive side of the X-Axis, and all negative values are grouped on the negative side of the X-Axis.
+
+### $PlatformShort$ Stacked Bar Chart Example 
+
+In this example of an $PlatformShort$ Stacked Bar Chart, we have a Numeric X Axis (bottom labels of the chart) and a Category Y Axis (left labels of the chart).
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+## $PlatformShort$ Stacked 100 Bar Chart
+
+The $PlatformShort$ Stacked 100 Bar Chart is identical to the $PlatformShort$ stacked bar chart in all aspects except in their treatment of the values on X-Axis (bottom labels of the chart). Instead of presenting a direct representation of the data, the stacked 100 bar chart presents the data in terms of percent of the sum of all values in a data point.
+
+### $PlatformShort$ Stacked 100 Bar Chart Example 
+
+In this example of a Stacked 100 Bar Chart, the Energy Product values are shown as a 100% value of all of the data in the fragments of the horizontal bars.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=8b193408-6f1f-42b1-812f-53783418bc68&mode=build

Reply:
**NEW EXAMPLE TO BE USED**

https://share.goabstract.com/1e0cf345-060c-49d3-8e04-6fb02aa21e2d?collectionLayerId=246e6c2d-4e14-4a31-b388-5ae88da227ac&mode=design